tools/junit_format_test_report_for_buildkite.rb

```diff
@@ -0,0 +1,12 @@
+module Minitest
```

Comment:
I think that it is appropriate that this file is under `ci`. 
https://github.com/rails/rails/tree/master/ci 

If it is likely to be used locally, this is fine. However, in that case I think that it is good to have an explanation in README. 
https://github.com/rails/rails/blob/master/tools/README.md

Reply:
@y-yagi I had been thinking of using this as:

```ruby
require_relative "../tools/test_common"
```

... and then having the "is this running on Buildkite?" condition inside that file (either to require an extra file, or just wrapped around the current content as an `if` block). There are other things in common between `*/test/test_helper.rb`, so it seems like it could be good to set up what's needed to start extracting them.

Reply:
@matthewd Oh, I see. It seems good :+1:

Reply:
> it seems like it could be good to set up what's needed to start extracting them.

I extracted as `tools/test_common.rb` 👍